packages/modal-ui/src/lib/components/WalletHome.tsx

```diff
@@ -98,25 +99,126 @@ export const WalletHome: React.FC<WalletHomeProps> = ({
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                      />
+                      <rect
+                        x="3.61108"
+                        y="3.33334"
+                        width="1.44444"
+                        height="1.33333"
+                        fill="#4C5155"
+                      />
+                      <rect
+                        x="3.61108"
+                        y="11.3333"
+                        width="1.44444"
+                        height="1.33333"
+                        fill="#4C5155"
+                      />
+                      <rect
+                        x="12.2778"
+                        y="3.33334"
+                        width="1.44445"
+                        height="1.33333"
+                        fill="#4C5155"
+                      />
+                    </svg>
+                  ) : null}
+                  {haveLink ? (
+                    <svg
+                      width="18"
+                      height="16"
+                      viewBox="0 0 18 16"
+                      fill="none"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        d="M13 8.66667V12.6667C13 13.0203 12.8478 13.3594 12.577 13.6095C12.3061 13.8595 11.9387 14 11.5556 14H3.61113C3.22804 14 2.86064 13.8595 2.58975 13.6095C2.31887 13.3594 2.16669 13.0203 2.16669 12.6667V5.33333C2.16669 4.97971 2.31887 4.64057 2.58975 4.39052C2.86064 4.14048 3.22804 4 3.61113 4H7.94447"
+                        stroke="#4C5155"
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                      />
+                      <path
+                        d="M10.8333 2H15.1666V6"
+                        stroke="#4C5155"
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                      />
+                      <path
+                        d="M7.22223 9.33333L15.1667 2"
+                        stroke="#4C5155"
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                      />
+                    </svg>
+                  ) : null}
+                </div>
                 <div className="icon">
                   <img src={iconUrl} alt={name} />
                 </div>
                 <div className="content">
                   <div className="title">{name}</div>
-                  <div className="description">{description}</div>
-                </div>
-                <div className="button-get">
-                  <button
-                    className="get-wallet"
-                    onClick={() => {
-                      goToWallet(module);
-                    }}
-                  >
-                    Get
-                  </button>
+                  <div className="type">{typeFullName}</div>
```

Comment:
In the latest design, the font-style is normal, not _italic_. See the **Flow** page in figma.